logzio-sender/src/test/java/io/logz/sender/InMemoryLogsBufferTest.java

```diff
@@ -0,0 +1,52 @@
+package io.logz.sender;
+
+import io.logz.sender.exceptions.LogzioParameterErrorException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.ScheduledExecutorService;
+
+public class InMemoryLogsBufferTest extends LogzioSenderTest {
```

Comment:
Where are all the test methods? In LogzioSenderTest?

Reply:
yes